log-factory/sinks/log-to-logbuffer/build.gradle

```diff
@@ -3,11 +3,8 @@ plugins {
   id 'io.deephaven.project.register'
 }
 
-configurations {
-  compileOnly.extendsFrom fishIo
-}
-
 dependencies {
+  compileOnly project(':Base')
```

Comment:
These `compileOnly` deps in the log-factory/X modules kind of surprise me.

Reply:
I agree; I don't think there is a technical reason why they need to be `compileOnly` at this level. Originally, I think this was done as a handwavy way to say that 3rd party code could depend on the logging API without pulling in the libraries themselves in the context of a "plugin" (sort of like how we have setup the authentication shadow jar stuff). If a 3rd party needs to have those semantics, they can declare their dependency as compileOnly themselves; ie, `compileOnly 'io.deephaven:deephaven-log-to-logbuffer:X.Y.Z'` for example.

Reply:
Filed https://github.com/deephaven/deephaven-core/issues/5732